stumpy/mstump.py

```diff
@@ -525,13 +615,34 @@ def multi_distance_profile(
         and the Euclidean distance, respectively. This parameter is ignored when
         `normalize == True`.
 
+    T_subseq_isconstant : numpy.ndarray or callable or list, default None
```

Comment:
Please replace the first line with "T_subseq_isconstant : numpy.ndarray, function, or list, default None"